cli/installer/util.go

```diff
@@ -63,10 +63,10 @@ func detectPkgManager() pkgManager {
 	}
 
 	switch true {
+	case commandExists("apt"):
```

Comment:
apt-get is deprecated

Reply:
it's not deprecated. it's intended for scripting, while `apt` is intended for manual user interaction. https://askubuntu.com/a/829866

https://manpages.ubuntu.com/manpages/xenial/man8/apt.8.html
https://linux.die.net/man/8/apt-get